tomviz/PresetDialog.h

```diff
@@ -0,0 +1,36 @@
+/* This source file is part of the Tomviz project, https://tomviz.org/.
+  It is released under the 3-Clause BSD License, see "LICENSE". */
+
+#ifndef tomvizPresetDialog_h
+#define tomvizPresetDialog_h
+
+#include "PresetModel.h"
+
+#include <QDialog>
+#include <QScopedPointer>
+
+namespace Ui {
+class PresetDialog;
+}
+
+namespace tomviz {
+
+class PresetDialog : public QDialog
+{
+  Q_OBJECT
+
+public:
+  explicit PresetDialog(QWidget* parent);
+  QString getName();
```

Comment:
We use the Qt naming convention - setter is called `setName(const QString& name)` and getter is called `name()`. Also prefer declaring the constructor, then the destructor, and then the other members. This may benefit from a little more detail, such as `presetName()` rather than just name.